docs/core/compatibility/core-libraries/6.0/numeric-format-parsing-handles-higher-precision.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Breaking change: Standard numeric format parsing precision"
+description: Learn about the .NET 6.0 breaking change in core .NET libraries where standard numeric format parsing now handles higher precisions.
+ms.date: 02/26/2021
+---
+# Standard numeric format parsing precision
+
+In .NET 6+, .NET supports precision up to <xref:System.Int32.MaxValue?displayProperty=nameWithType> when formatting numbers as strings using `ToString` and `TryFormat`.
+
+## Change description
+
+In previous .NET versions, the standard numeric format parsing logic is limited to a precision of 99 or less. Some numeric types have more precision, but `ToString(string format)` does not expose it correctly. If you specify a precision greater than 99, for example, `32.ToString("C100")`, the format string is interpreted as a custom numeric format string instead of "currency with precision 100". In custom numeric format strings, characters are interpreted as character literals. In addition, a format string that contains an invalid format specifier (that is, the character at the beginning of the string) is interpreted differently depending on the precision value. `H99` throws a <xref:System.FormatException> for the invalid format specifier, while `H100` is interpreted as a custom numeric format string.
+
+Starting in .NET 6, .NET supports precision up to <xref:System.Int32.MaxValue?displayProperty=nameWithType>. A format string that consists of a valid format specifier with any number of digits is interpreted as a standard numeric format string with precision. If you specify a precision that's greater than <xref:System.Int32.MaxValue?displayProperty=nameWithType>, a <xref:System.FormatException> is thrown. This change was implemented in the parsing logic that affects all numeric types.
+
+The following table shows how the behavior for various format strings changed.
+
+| Format string | Previous behavior | .NET 6+ behavior |
```

Comment:
@pgovind Can you check these examples?

Reply:
The last one `H100` needs to be changed. `H` is an invalid standard format specifier, so it would throw a `FormatException` (same as `H99`). Up to you if you want to keep both `H99` and `H100` or pick one :)